Overflow in pool reward calculation

There's a panic due to an overflow when calculating rewards.

The offending line is `scaledPoolReward, err := safemath.SafeMul(result, performanceFactor)` but judging by the error message, the `result` that's being passed in is already greater than it should be. So the problem most likely lies somewhere earlier in that code path.

```
panic: integer under- or overflow: 12561066653626084440 * 7: failed to calculate pool reward without fixed costs due to overflow for slot 64: failed to calculate pool rewards for account 0x6aee704f25558e8aa7630fed0121da53074188abc423b3c5810f80be4936eb6e

goroutine 209 [running]:
github.com/iotaledger/iota-core/pkg/protocol/sybilprotection/sybilprotectionv1/performance.(*Tracker).ApplyEpoch.func1({0x6a, 0xee, 0x70, 0x4f, 0x25, 0x55, 0x8e, 0x8a, 0xa7, 0x63, ...}, ...)
	/scratch/iota-core/pkg/protocol/sybilprotection/sybilprotectionv1/performance/performance.go:253 +0x3f3
github.com/iotaledger/hive.go/ds/shrinkingmap.(*ShrinkingMap[...]).ForEach(0x2d38160, 0xc0019e7618)
	/go/pkg/mod/github.com/iotaledger/hive.go/ds@v0.0.0-20231127134220-90b88e35bdb2/shrinkingmap/shrinkingmap.go:165 +0x278
github.com/iotaledger/iota-core/pkg/core/account.(*Accounts).ForEach(...)
	/scratch/iota-core/pkg/core/account/accounts.go:125
github.com/iotaledger/iota-core/pkg/protocol/sybilprotection/sybilprotectionv1/performance.(*Tracker).ApplyEpoch(0xc000266680, 0x0, 0xc00171db40)
	/scratch/iota-core/pkg/protocol/sybilprotection/sybilprotectionv1/performance/performance.go:214 +0x369
github.com/iotaledger/iota-core/pkg/protocol/sybilprotection/sybilprotectionv1.(*SybilProtection).CommitSlot(0xc0007d1760, 0x40)
	/scratch/iota-core/pkg/protocol/sybilprotection/sybilprotectionv1/sybilprotection.go:190 +0x2d2
github.com/iotaledger/iota-core/pkg/protocol/engine/notarization/slotnotarization.(*Manager).createCommitment(0xc00001c000, 0x40)
	/scratch/iota-core/pkg/protocol/engine/notarization/slotnotarization/manager.go:215 +0x559
github.com/iotaledger/iota-core/pkg/protocol/engine/notarization/slotnotarization.(*Manager).tryCommitSlotUntil(0xc00001c000, 0x47)
	/scratch/iota-core/pkg/protocol/engine/notarization/slotnotarization/manager.go:179 +0x92
github.com/iotaledger/iota-core/pkg/protocol/engine/notarization/slotnotarization.(*Manager).tryCommitUntil(0xc00001c000, 0x47)
	/scratch/iota-core/pkg/protocol/engine/notarization/slotnotarization/manager.go:108 +0x45
github.com/iotaledger/iota-core/components/protocol.provide.func1.NewProvider.func14.1.1(0xc0019e7ed8?)
	/scratch/iota-core/pkg/protocol/engine/notarization/slotnotarization/manager.go:72 +0x131
github.com/iotaledger/hive.go/runtime/event.(*Event1[...]).Trigger.func1.1()
	/go/pkg/mod/github.com/iotaledger/hive.go/runtime@v0.0.0-20231128121006-331a9e522dfe/event/events.go:86 +0x1f
github.com/iotaledger/hive.go/runtime/workerpool.(*Task).run(0xc001b20f00)
	/go/pkg/mod/github.com/iotaledger/hive.go/runtime@v0.0.0-20231128121006-331a9e522dfe/workerpool/task.go:48 +0x72
github.com/iotaledger/hive.go/runtime/workerpool.(*WorkerPool).workerReadLoop(0xc0002a4b60)
	/go/pkg/mod/github.com/iotaledger/hive.go/runtime@v0.0.0-20231128121006-331a9e522dfe/workerpool/workerpool.go:190 +0x25
github.com/iotaledger/hive.go/runtime/workerpool.(*WorkerPool).worker(0xc0002a4b60)
	/go/pkg/mod/github.com/iotaledger/hive.go/runtime@v0.0.0-20231128121006-331a9e522dfe/workerpool/workerpool.go:170 +0x50
created by github.com/iotaledger/hive.go/runtime/workerpool.(*WorkerPool).startWorkers in goroutine 1
	/go/pkg/mod/github.com/iotaledger/hive.go/runtime@v0.0.0-20231128121006-331a9e522dfe/workerpool/workerpool.go:162 +0x25
```

--- a/pkg/testsuite/testsuite.go
+++ b/pkg/testsuite/testsuite.go
@@ -47,7 +47,7 @@ func DefaultProtocolParameterOptions(networkName string) []options.Option[iotago
 			100,
 			100,
 		),
-		iotago.WithRewardsOptions(8, 8, 31, 1154, 2, 1),
+		iotago.WithRewardsOptions(8, 8, 11, 1154, 2, 1),
 		iotago.WithStakingOptions(1, 100, 1),
 
 		iotago.WithTimeProviderOptions(

